.github/workflows/build.yml

```diff
@@ -55,8 +95,18 @@ jobs:
           chmod +x ./scripts/setup_west_workspace.sh
           ./scripts/setup_west_workspace.sh
 
-      - name: Build app
+      - name: Builds
         run: |
+          set -e
```

Comment:
Do we want this? I'm not sure how `set -e` behaves in GitHub Actions. Will it force-quit all queued and running jobs (does it cancel runs of other build jobs, like `pay`, if `obc` fails)?